doc/source/whatsnew/v1.1.1.rst

```diff
@@ -15,7 +15,7 @@ including other versions of pandas.
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
 
--
+- Bug in :class:`pandas.core.groupby.RollingGroupby` where ``__getitem__`` would not select the specified column (:issue:`35486`)
```

Comment:
Minor nit but is there a way to make this more user-friendly? Fair to say something like `Bug in ... where column selection was being ignored`?

Reply:
Sure, ya that phrasing would be more user friendly